install.sh

```diff
@@ -55,55 +57,85 @@ case $install_dvm in
 esac
 
 echo "Cloning branch: $branch"
+sleep 2
```

Comment:
Is there any reason for including these `sleep` statements? It will only make the script longer to run. If we can remove that would be better.

Reply:
One of the reasons people get lost is because they don't  know what happened. So those sleep doesn't slow down the process that much and help the user see what's happening or where exactly the script is failing. Thought it was a nice addition. it's like 15s more tops haha

Reply:
ok but usually if the script fails then the user can just scroll up and check where and why it failed, I'll merge in for now and if this is an issue we can always remove the `sleep`